package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "homepage": "https://github.com/davidchambers/string-format",
   "bugs": "https://github.com/davidchambers/string-format/issues",
-  "license": "WTFPL",
+  "license": "WTFPL or MIT",
```

Comment:
Based on the [npm documentation][1] it seems the value should be `"(WTFPL OR MIT)"`:

```javascript
> var spdx = require('spdx')

> spdx.valid('WTFPL or MIT')
null

> spdx.valid('(WTFPL OR MIT)')
true
```


[1]: https://docs.npmjs.com/files/package.json#license